### PR TITLE
Added upgrade instuctions

### PR DESCRIPTION
**Pull request by starred-gijs**
Please review and improve

### CODE REVIEW

File: UPGRADE-0.4.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+Since version 0.4.0 this package uses the transport provided by [Symfony Amazon SQS Messenger](https://symfony.com/doc/current/messenger.html#amazon-sqs) 
+and comes with these potential BC breaks:
+
+Message Headers
+---------------
+>  This does not affect transports that have the default `PhpSerializer` 
+
+Previously the message headers were combined in a [single MessageAttribute](https://github.com/brefphp/symfony-messenger/blob/0.3.4/src/Service/Sqs/SqsTransport.php#L46),
+now each header is a [separate MessageAttribute](https://github.com/symfony/amazon-sqs-messenger/blob/5.x/Transport/Connection.php#L316).
```
Comment (Nyholm) [inline UPGRADE-0.4.md:12]: Lets link to a tag instead

File: UPGRADE-0.4.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+>  This does not affect transports that have the default `PhpSerializer` 
+
+Previously the message headers were combined in a [single MessageAttribute](https://github.com/brefphp/symfony-messenger/blob/0.3.4/src/Service/Sqs/SqsTransport.php#L46),
+now each header is a [separate MessageAttribute](https://github.com/symfony/amazon-sqs-messenger/blob/5.x/Transport/Connection.php#L316). 
+This means that the SQS records are incompatible between bref/symfony-messenger 0.3.x and 0.4.0.
+
+
+Fifo Queues
+-----------
+The setup for Fifo queues has changed; from transport configuration to a AmazonSqsFifoStamp
```
Comment (Nyholm) [inline UPGRADE-0.4.md:18]: Lets do full namespace to `AmazonSqsFifoStamp`

**Review by Nyholm — COMMENTED** [review]
Good. Make sure to write a entry in the Changelog.md too referring to this document

**Review by Nyholm — APPROVED** [review]

**Review by mnapoli — APPROVED** [review]
Thanks!

**Pull request merged**